frontend/src/routes/routes.js

```diff
@@ -19,7 +19,9 @@ const routes = [
     },
   },
   {
-    path: '/send',
+    // userIdentifier can be username, email or gradidoID
+    // communityIdentifier can be community name or community UUID
+    path: '/send/:communityIdentifier?/:userIdentifier?',
```

Comment:
wie geht das framework mit `undefined`/`null` für `communityIdentifier` um, wenn `userIdentifier` initialisiert ist?
steht dann in der URL `send//bibi` oder so?

Reply:
https://router.vuejs.org/guide/essentials/route-matching-syntax.html#Optional-parameters

Das ist ein Router, er reagiert auf die url die eingegeben wurde. Er erzeugt keine url.
Die Parameter sind beide als optional gekennzeichnet.
Es kann also all das benutzt werden:
- /send
- /send/gradido-community
- /send/gradido-community/bibi 
benutzt werden. 

Und es wird immer auf die Send-Page weitergeleitet.

Reply:
Der erste Parameter nach /send wird immer als community-identifier interpretiert. 
Ein community-identifier kann alleine benutzt werden um die Ziel-Community Vorauszuwählen.
Aber ein Benutzer kann nicht alleine genutzt werden, da ohne Community es nicht eindeutig ist von welcher Community der User ist.